zookeeper-server/zookeeper-server-common/src/main/java/com/yahoo/vespa/zookeeper/Reconfigurer.java

```diff
@@ -111,6 +115,9 @@ private void reconfigure(ZookeeperServerConfig newConfig) {
                 now = Instant.now();
             }
         }
+
+        // Reconfiguration failed
+        return false;
```

Comment:
Instead of returning boolean, how about calling

```
server.shutdown();
Process.logAndDie("Reconfiguration did not complete within timeout " + reconfigTimeout(newServers.size()) + ". Forcing shutdown")
```

here? Could extract a variable for reconfigTimeout as it's used earlier in this method.

Reply:
Yes, that sounds, better, thanks